x/launch/keeper/msg_server_request_add_vested_account_test.go

```diff
@@ -0,0 +1,93 @@
+package keeper
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/spn/testutil/sample"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func TestMsgRequestAddVestedAccount(t *testing.T) {
+	var (
+		invalidChain, _           = sample.ChainID(0)
+		addr1                     = sample.AccAddress()
+		addr2                     = sample.AccAddress()
+		addr3                     = sample.AccAddress()
+		k, _, srv, _, sdkCtx, cdc = setupMsgServer(t)
+		ctx                       = sdk.WrapSDKContext(sdkCtx)
+		chains                    = createNChain(k, sdkCtx, 4)
+	)
+	tests := []struct {
+		name string
+		msg  types.MsgRequestAddVestedAccount
+		want uint64
+		err  error
+	}{
+		{
+			name: "invalid chain",
+			msg: types.MsgRequestAddVestedAccount{
+				ChainID: invalidChain,
```

Comment:
Same feedback as for `MsgRequestRemoveAccount`

Let's define all the fields for each message test